jekyll/_cci2/release/set-up-a-release-environment.adoc

```diff
@@ -0,0 +1,111 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Set up a release environment - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to set up a new CircleCI release environment and install the CircleCI release agent.
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+Visualise and control deployments to your Kubernetes cluster with CircleCI release environments. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will set up an integration between CircleCI and your Kubernetes cluster. Once configured, release status can be reported to the CircleCI web app, and the following commands can be executed in your cluster by the release agent via controls in the CircleCI UI:
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+In order to set up a release environment with CircleCI you will need to ensure the following prerequisites are fulfilled:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A Kubernetes cluster
+* A project building on CircleCI that deploys to your Kubernetes cluster. See the xref:../create-project#[Create a project] page for steps.
+* You must have write access to the project associated with the component being deployed. For full details of roles and permissions for GitLab and GitHub App integrations, see the xref:../roles-and-permissions-overview.adoc#[Roles and permissions overview]. If your integration is through Bitbucket or the GitHub OAuth app, you will need to be an org admin in Bitbucket/GitHub respectively. To find out which GitHub integration you have, see the xref:../github-apps-integration.adoc#[GitHub App integration] page
+
+CircleCI supports Kubernetes, Helm, and Argo Rollouts. The supported versions are listed below:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|System
+|Supported versions
+
+|Kubernetes
+a| 1.28.0 +
```

Comment:
We're gonna have to find a way to expand this list, without putting us on the hook for supporting them all. eg: tested on vs supported on.  Because otherwise this will be a blocker for users, since i bet many are not always on the latest/greatest (eg: we are only now doing 1.26 on our own prod k8s. ) @alexazl @ryanbarkley

Reply:
What if we say the listed versions are ones we've tested against and older versions may still work (though not guaranteed)?

Reply:
Yeah! We have similar for server actually: https://circleci.com/docs/server/v4.2/overview/circleci-server-overview/#platforms